sap/adt/datapreview.py

```diff
@@ -92,7 +93,7 @@ def parse_freestyle_table(freestyle_table_xml, rows):
     """Converts XML results into Python representation"""
 
     xml_handler = FreeStyleTableXMLHandler(rows)
-    xml.sax.parseString(freestyle_table_xml, xml_handler)
+    xml.sax.parseString(wrap_xml_data_content_with_cdata(freestyle_table_xml), xml_handler)
```

Comment:
I think this is no longer necessary, isn't it?

Reply:
It is the most important and must remain there. If the string, that contains `&`, is not wrapped with `<![CDATA[...]]>` then the XML parser splits the string by the `&` (XML special character) and we will see `London` only (as a string behind the last `&`). Such string is parsed well only if it is wrapped by `<![CDATA[...]]>`

Reply:
It was a bug. I am always forgetting that the method characters does not necessarily get the full contents of text nodes.